pkg/gui/containers_panel.go

```diff
@@ -496,6 +496,27 @@ func (gui *Gui) handleContainersRemoveMenu(g *gocui.Gui, v *gocui.View) error {
 	return gui.createMenu("", options, len(options), handleMenuPress)
 }
 
+func (gui *Gui) handleContainerPause(g *gocui.Gui, v *gocui.View) error {
+	container, err := gui.getSelectedContainer()
+	if err != nil {
+		return nil
+	}
+
+	return gui.WithWaitingStatus(gui.Tr.PausingStatus, func() error {
```

Comment:
I'd extract this part out into its own function and call that from handleServicePause so that we're not duplicating the logic

Reply:
I've pushed it in a new commit so when you find the time you can easily test out the automatic recognizing of pausedness (I know it's not a word, but I do think it should be, haters be damned).